### DISCUSSION TITLE
Feature Request: Support for clob datatype

### DISCUSSION THREAD

**Discussion by dantefff** [duckdb/duckdb#12407] [Ideas]
I'm trying to use duckdb as development in memory database for a spring-hibernate app. When hibernate interprets some of my entities with attributes of type string and anotated as @Lob tries to create database with columns of clob type.
Then I receive a SqlException saying 
_java.sql.SQLException: Catalog Error: Type with name clob does not exist!
Did you mean "blob"?_

App uses postgres as in-production database so duckdb sounded as a good development in-memory option, as it claims to be closely following the conventions of the PostgreSQL dialect but not in this case.

**Comment by dantefff** [comment]
This is solved by using as hibernate.dialect org.hibernate.dialect.PostgreSQLDialect. Is there a duckdb dialect available?

**Comment by soerenwolfers** [comment]
Is duckdb's VARCHAR not performant enough?
